Jenkinsfile

```diff
@@ -985,7 +985,8 @@ pipeline {
                     steps {
                         functionalTest inst_repos: el7_daos_repos(),
                                        inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                                                  ' ' + functional_rpms +
+                                                  ' python2-tabulate '
```

Comment:
This should probably be an RPM `Requires:`.

Reply:
In [daos.spec](https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec)?

Reply:
Yes.  Since this is something anyone wanting to run tests from daos-tests will need to have installed, yes?

Reply:
60c8ef19a03e1a7e5aaee2a59e7c2ab60f6d9362